realtime/redisPublisher.js

```diff
@@ -134,6 +141,39 @@ function publishSample(sampleInst, subjectModel, event, aspectModel) {
   });
 } // publishSample
 
+/**
+ * Publish a sample when nothing changed except last update timestamp.
+ *
+ * @param  {Object} sample - The sample to be published
+ * @returns {Promise} - which resolves to the object that was published
+ */
+function publishSampleNoChange(sample) {
+  const s = {
+    name: sample.name,
+    status: sample.status, // for socket.io perspective filtering
+    absolutePath: sample.absolutePath, // used for persp filtering
+    updatedAt: sample.updatedAt,
+    subject: {
+      absolutePath: sample.absolutePath,
+      tags: sample.subjectTags, // for socket.io perspective filtering
+    },
+    aspect: {
+      name: sample.aspectName, // for socket.io perspective filtering
+      tags: sample.aspectTags, // for socket.io perspective filtering
```

Comment:
Should we maybe strip out these fields in the emitter or perspective so it's clear to the lens that it shouldn't rely on this object for any data other than updatedAt?

Reply:
I like that idea--let me take a stab at that!!!

The first benefit is as you wrote, but another benefit is smaller event data means less work serializing, and as we saw today, that takes up a lot of our time!